engines/python/setup/djl_python/deepspeed.py

```diff
@@ -210,6 +208,10 @@ def create_model_pipeline(self):
                 self.model_id_or_path,
                 low_cpu_mem_usage=self.low_cpu_mem_usage,
                 **kwargs)
+        if self.data_type:
+            self.ds_config["dtype"] = self.data_type
+        elif model.dtype:
```

Comment:
just do else. There should be no case that model doesn't have a dtype.

Reply:
oops 😆